composer.json

```diff
@@ -9,32 +9,30 @@
     "require": {
         "ext-dom": "*",
         "ext-libxml": "*",
-        "city-of-helsinki/helfi-kymp-district-project-search": "0.4.0",
+        "city-of-helsinki/helfi-kymp-district-project-search": "0.4.2",
         "composer/installers": "^2.0",
         "cweagans/composer-patches": "^1.6.7",
         "drupal/core-composer-scaffold": "~9.4.5",
         "drupal/core-recommended": "~9.4.5",
         "drupal/filelog": "^2.1",
         "drupal/hdbt": "^4.0",
         "drupal/hdbt_admin": "^1.0",
-        "drupal/helfi_ahjo": "^1.0",
+        "drupal/helfi_api_base": "dev-UHF-7265",
```

Comment:
The composer dependencies need to be addressed before merging to main. 
`"drupal/helfi_api_base": "dev-UHF-7265",`